tools/gstf_preparation/gstf_preparation.py

```diff
@@ -122,6 +126,12 @@ def feature_to_dict(cols, parent_dict=None):
 def add_gene_to_dict(cols, species, gene_dict):
     global gene_count
     gene = feature_to_dict(cols)
+    if 'biotype' in gene and gene['biotype'] != 'protein_coding':
+        return
```

Comment:
At the moment this is filtering of gene and transcripts is applied only to GFF3 files, not gene features in JSON format. Do we want to extend it to JSON files too?